src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs

```diff
@@ -42,7 +42,6 @@ internal abstract partial class CSharpAttributeData : AttributeData
         public new abstract SyntaxReference? ApplicationSyntaxReference { get; }
 
         // Overridden to be able to apply MemberNotNull to the new members
```

Comment:
It looks like we should also delete the comment, or possibly delete this member declaration entirely.